etna/transforms/holiday.py

```diff
@@ -0,0 +1,56 @@
+import holidays
+import numpy as np
+import pandas as pd
+
+from etna.transforms.base import Transform
+
+
+class HolidayTransform(Transform):
+    """HolidayTransform generates series that indicates holidays in given dataframe. Creates column 'regressor_holidays'."""
+
+    def __init__(self, iso_code: str = "RUS"):
```

Comment:
you need to save `iso_code` in `self.iso_code` for correct `__repr__`

Reply:
moreover, all the transforms in ETNA have arg `output_column: Optional[str] = None`
if `output_column is not None` the transform creates the columns with  this name (not `regressor_holidays`), otherwise it creates column with name `f"regressor_{self.__repr__}"` (prefix `regressor_` is added because `HolidayTransform` can generate future features)